jerry-ext/handler/handler-print.c

```diff
@@ -56,24 +56,24 @@ jerryx_handler_print (const jerry_value_t func_obj_val, /**< function object */
 
     if (!jerry_value_is_error (str_val))
     {
-      if (arg_index != 0)
-      {
-        jerryx_port_handler_print_char (' ');
-      }
-
+      const jerry_size_t substr_max_size = 256;
       jerry_size_t substr_size;
       jerry_length_t substr_pos = 0;
-      jerry_char_t substr_buf[256];
+      jerry_char_t substr_buf[substr_max_size];
```

Comment:
Plase use `JERRY_VLA` to fix the windows build.

Reply:
But this is a fixed size array :(

Reply:
What about using #define instead of a const variable?